generator/statistics.cpp

```diff
@@ -13,35 +13,37 @@
 #include <iterator>
 
 using namespace feature;
-using namespace std;
```

Comment:
Put using namespace under namespace stats here.
Also, I don't mind to have using std under stats, but it's up to you.